src/common/layout/PageLayout.module.css

```diff
@@ -15,4 +23,27 @@ header {
 
 .youthBackground {
     background-color: #ffc61e;
-}
\ No newline at end of file
+}
+
+button {
+    background-color: #0000bf;
+    color: #ffffff;
+    height: 50px;
+    width: 400px;
+}
+
+@media (max-width: 1200px) {
+    button {
+        width: 300px;
+    }
+  }
+
+  @media (max-width: 320px) {
```

Comment:
Since 414px is used in other places for the smallest breakpoint, I would suggest you use it here too for consistency.